packages/@glimmer-workspace/integration-tests/lib/setup-harness.ts

```diff
@@ -23,6 +23,18 @@ export async function setupQunit() {
     tooltip: 'CI mode makes tests run faster by sacrificing UI responsiveness',
   });
 
+  QUnit.config.urlConfig.push({
```

Comment:
nice -- tbh, I think we should document this on the ember guides -- gonna open an issue over there